### PR TITLE
Docker Volume Drivers

### PR DESCRIPTION
**Pull request by pietromenna**
This commit adds the functionality to use Docker Volume Drivers.

### CODE REVIEW

File: website/source/docs/drivers/docker.html.md [hunk 1]
```diff
@@ -201,6 +201,26 @@ The `docker` driver supports the following configuration in the job spec:
...
+        # Use absolute paths to mount arbitrary paths on the host
+        "/path/on/host:/path/in/container",
+
+        # Use relative paths to rebind paths already in the allocation dir
+        "relative/to/task:/also/in/container",
+
+        # Use named volume created outside nomad.
+        "name-of-the-volume:/path/in/container"
+      ]
+      volume_driver = "flocker"
```
Comment (schmichael) [inline website/source/docs/drivers/docker.html.md]: Add a comment to draw attention to the `volume_driver` field being described.

File: website/source/docs/drivers/docker.html.md [hunk 2]
```diff
@@ -201,6 +201,26 @@ The `docker` driver supports the following configuration in the job spec:
...
+  well as regular paths.
+
+  ```hcl
+    config {
+      volumes = [
+        # Use absolute paths to mount arbitrary paths on the host
+        "/path/on/host:/path/in/container",
+
+        # Use relative paths to rebind paths already in the allocation dir
+        "relative/to/task:/also/in/container",
```
Comment (schmichael) [inline website/source/docs/drivers/docker.html.md]: Remove these other examples that are already included above as I think the `name-of-the-volume` example is the important one when showing how to use `volume_driver`

**Comment by pietromenna** [comment]
Hi @schmichael ,

I just incorporated the changes you suggested.

Thanks!

**Comment by schmichael** [comment]
Perfect! Thanks!

**Pull request merged**